### ISSUE TITLE
I have a question about the combo box keyboard interaction.

### ISSUE THREAD

**Issue by mulder21c** [w3c/aria-practices#2050]
in [Note on Grid Popup Keyboard Interaction](https://w3c.github.io/aria-practices/#h-note-2),

> If every cell contains a different suggested value:
>  - Selection follows focus so that the cell containing focus is selected.
>  - Horizontal arrow key navigation typically wraps from one row to another.
>  - Vertical arrow key navigation typically wraps from one column to another.

`Horizontal arrow key navigation typically wraps from one row to another.` is right?

As far as I know, the horizontal key is ← or →, but I think it should be moved between columns.

If I'm mistaken, I'd appreciate it if you let me know.

**Comment by mcking65** [comment]
>As far as I know, the horizontal key is ← or →, but I think it should be moved between columns.

You are correct, left/right navigate horizontally across rows from one cell to another.

The text you are referencing is describing the wrapping behavior at the beginning or end of a row. When the last cell in a row has focus, instead of stopping, right will typically move to the first value in the next row. At the beginning of a row, left will move to the last value in the previous row.

Does that answer your question?

We will add some more text to explain that.
